receiver/gitproviderreceiver/internal/scraper/githubscraper/helpers.go

```diff
@@ -312,18 +313,77 @@ func getAge(start time.Time, end time.Time) int64 {
 func (ghs *githubScraper) getCVEs(
 	ctx context.Context,
 	client graphql.Client,
+	rClient *github.Client,
 	repo string,
 ) (map[metadata.AttributeCveSeverity]int64, error) {
-	c, err := getRepoCVEs(ctx, client, ghs.cfg.GitHubOrg, repo)
-	if err != nil {
-		return nil, err
+	d := ghs.getDbotAlerts(ctx, client, repo)
```

Comment:
```suggestion
	d := ghs.getDbotAlerts(ctx, gClient, repo)
```

Reply:
Both calls have been updated to use their respective clients with newer naming. Marking as resolved